### PR TITLE
Issue: 23655 Fix error when this._preview is null

### PR DESCRIPTION
**Pull request by YugasVasyl**
https://github.com/angular/components/issues/23655

### CODE REVIEW

**Comment by YugasVasyl** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fangular%2Fcomponents%2Fpull%2F23656) for more info**.

@googlebot I signed it!

File: src/cdk/drag-drop/drag-ref.ts [hunk 1]
```diff
@@ -1097,7 +1097,9 @@ export class DragRef<T = any> {
         const handler = ((event: TransitionEvent) => {
           if (!event || (_getEventTarget(event) === this._preview &&
               event.propertyName === 'transform')) {
-            this._preview.removeEventListener('transitionend', handler);
+            if (this._preview) {
+              this._preview.removeEventListener('transitionend', handler);
```
Comment (crisbeto) [inline src/cdk/drag-drop/drag-ref.ts:1101]: nit: can be reduced to `this._preview?.removeEventListener('transitionend', handler);`

**Review by crisbeto — APPROVED** [review]
LGTM

**Comment by crisbeto** [comment]
@YugasVasyl the CI is failing because the commit message isn't formatted correctly. It should be changed to something like:

```
fix(cdk/drag-drop): error when preview is destroyed before animation completes
```

**Comment by YugasVasyl** [comment]
@crisbeto not sure how to edit pushed commit, created new PR here https://github.com/angular/components/pull/23662

**Comment by crisbeto** [comment]
Closing in favor of https://github.com/angular/components/pull/23662.

**Pull request closed without merging**